app/amity.py

```diff
@@ -207,54 +287,271 @@ def load_people(self, filename):
                         person_data = line.split()
                         first_name = person_data[0]
                         last_name = person_data[1]
-                        full_name = first_name + " " + last_name
+                        empno = first_name + last_name
                         job_type = person_data[2]
                         try:
                             accomodation = person_data[3]
                         except Exception:
                             accomodation = 'N'
 
-                        self.add_person(full_name, job_type, accomodation)
+                        self.add_person(empno.upper(), first_name.upper(), last_name.upper(), job_type.upper(),
+                                        accomodation.upper())
 
                     return "File data added successfully"
 
         else:
-            print("please provide a text file name")
+            return "Please provide valid a text file name !"
 
     def print_allocations(self, filename=None):
-        """prints all allocations"""
-        print("-" * 30 + "\n" + "AMITY OFFICE ALLOCATIONS\n" + "-" * 30 + "\n")
-        for office, names in self.office_allocations.items():
-            print(office.upper())
-            print("-" * 30)
-            print("\n")
-            for name in names:
-                print(name)
-                print("\n")
+        """Prints a list of allocations onto the screen. Specifying the optional -o option here outputs the registered
+        allocations to a txt file"""
+        if len(self.office_allocations) or len(self.lspace_allocations) != 0:
```

Comment:
an empty list by default is falsy ,  so no need to check for length to confirm if its true or false